src/extension/rn-extension.ts

```diff
@@ -20,7 +21,8 @@ import {OutputChannelLogger} from "./outputChannelLogger";
 
 /* all components use the same packager instance */
 const globalPackager = new Packager(vscode.workspace.rootPath);
-const commandPaletteHandler = new CommandPaletteHandler(vscode.workspace.rootPath, globalPackager);
+const packagerStatusIndicator = new PackagerStatusIndicator();
```

Comment:
No, unless we explicitly call UpdateStatus (which in turn calls statusBarItem.show()), this will be hidden :)